ci/slab.toml

```diff
@@ -1,16 +1,16 @@
 [profile.cpu-big]
 region = "eu-west-3"
-image_id = "ami-0ab73f5bd11708a85"
+image_id = "ami-051942e4055555752"
```

Comment:
updated with openssl properly installed and update packages I'm guessing ?

Reply:
yes!